tests/unauthorized/filesTest.js

```diff
@@ -1,34 +1,129 @@
-describe('Unauthorized: Currently testing files management,', function () {
-// CURRENT TIME
-  var timeRightNow = new Date().getTime()
-  var OwnCloud = require('../../src')
-  var config = require('../config/config.json')
+fdescribe('Unauthorized: Currently testing files management,', function () {
+  const OwnCloud = require('../../src')
+  const config = require('../config/config.json')
 
   // LIBRARY INSTANCE
-  var oc
+  let oc
 
   // PACT setup
   const Pact = require('@pact-foundation/pact-web')
+  const { setGeneralInteractions, invalidAuthHeader, origin } = require('../pactHelper.js')
   const provider = new Pact.PactWeb()
-  const { setGeneralInteractions } = require('../pactHelper.js')
+  const requestHeaderWithInvalidAuth = {
+    authorization: invalidAuthHeader,
+    Origin: origin
+  }
+  const expectedUnAuthorizedMessage = 'Username or password was incorrect, Username or password was incorrect'
+  const incorrectAuthorizationXmlResponseBody = '<?xml version="1.0" encoding="utf-8"?>\n' +
+    '<d:error xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns">\n' +
+    '  <s:exception>Sabre\\DAV\\Exception\\NotAuthenticated</s:exception>\n' +
+    '  <s:message>Username or password was incorrect, Username or password was incorrect</s:message>\n' +
+    '</d:error>'
+  const webdavFilesResponseHeader = {
+    'Access-Control-Allow-Origin': origin,
+    'Content-Type': 'application/xml; charset=utf-8',
+    'Access-Control-Allow-Methods': 'GET,OPTIONS,POST,PUT,DELETE,MKCOL,PROPFIND,PATCH,PROPPATCH,REPORT,HEAD,COPY,MOVE'
```

Comment:
why not using `accessControlAllowMethods`?

Reply:
done!